libafl_sugar/Cargo.toml

```diff
@@ -27,9 +27,9 @@ aarch64 = ["libafl_qemu/aarch64"] # build qemu for aarch64
 pyo3-build-config = { version = "0.16", optional = true }
 
 [dependencies]
-libafl = { path = "../libafl", version = "0.8.1" }
-libafl_targets = { path = "../libafl_targets", version = "0.8.1" }
-libafl_qemu = { path = "../libafl_qemu", version = "0.8.1" }
+libafl = { path = "../libafl", version = "0.8.2" }
```

Comment:
@andreafioraldi no need to have versions here, the path is enough

Reply:
No for publication its not